src/continuum_absorption/absorption_H.jl

```diff
@@ -286,7 +286,6 @@ Hminus_ff = bounds_checked_absorption(_Hminus_ff,
 
 function _H2plus_bf_and_ff(ν::Real, T::Real, nH_I_div_partition::Real, nH_II::Real)
```

Comment:
So I don't think you deleted the old implementation of `_H2plus_bf_and_ff`. I think you should probably delete the old definition (especially since the code used in the unit tests is a direct transcription of the data that this polynomial fits).

Honestly, I'm surprised that Julia hasn't complained about having multiple definitions for `_H2plus_bf_and_ff` with the same function signature.?